### DISCUSSION TITLE
Rechteck-Abfrage von GeoServer-WFS-Diensten liefert nur 1 Treffer

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#29] [Q&A] [answered]
Wird mit dem Rechteck-Werkzeug ein "identify" über mehrere Objekte in einem GeoServer-WFS-Dienst aufgezogen, wird nur ein (1) Treffer dargestellt.
In den Entwicklertools wird folgende Meldung angezeigt:
{
  "exception": "Unhandeld response type: E.Standard.WebMapping.Core.ServiceResponses.ServiceResponse",
  "exception_type": "exception",
  "success": false
}
![rechteckabfrage_1](https://user-images.githubusercontent.com/100088983/214032882-27b28e39-157c-42d2-a205-f45957ebc466.png)
![rechteckabfrage_2](https://user-images.githubusercontent.com/100088983/214032885-fdcca280-4b7d-4516-9108-ae2097df98dd.png)

Dasselbe Phänomen tritt auch bei anderen GeoServer-WFS-Diensten auf (z.B: biotope)

**Comment by gis-eni** [comment]
Kann ich leider nicht nachvollziehen.
Da bräuchte ich mehr Details: Dienst, CMS, ...
Kannst mit das was zur Verfügung stellen? (per MAIL!!!)

**Comment by faulgurke** [comment, marked as answer]
Das Problem resultiert nicht wie zunächst vermutet aus der Vertauschung von Rechts- und Hochwert, sondern aus einem Kodierungsproblem bei der GeoServer Installation auf Windows (unter Linux tritt das Problem nicht auf).
In den Global-Settings von GeoServer war „UTF-8“ eingestellt. Dies hat beim Abfragen von Selektionen über die Rechteckauswahl zu Fehlern geführt. Die Umstellung der Codierung auf „ISO-8859-1“ hat dann das Problem gelöst.
